### PR TITLE
Add rollup-plugin-less typings for version 1.1

### PR DESCRIPTION
**Pull request by tristan-f-r**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/rollup-plugin-less/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Type definitions for rollup-plugin-less 1.1
+// Project: https://github.com/xiaofuzi/rollup-plugin-less##readme
+// Definitions by: Tristan <https://github.com/LeoDog896>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
```
Comment (peterblazejewicz) [inline types/rollup-plugin-less/index.d.ts:5]:
```suggestion
```
it's a history, 3.8 gets tested here (or newer)

File: types/rollup-plugin-less/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+/// <reference types="node" />
+/// <reference types="jsdom" />
```
Comment (peterblazejewicz) [inline types/rollup-plugin-less/index.d.ts:8]:
```suggestion
```
do you need this?

Comment (LeoDog896) [reply types/rollup-plugin-less/index.d.ts:7]: It didn't seem to work without it -- I'll test it locally first.

Comment (peterblazejewicz) [reply types/rollup-plugin-less/index.d.ts:7]: no worries, that can be required, let me know

Comment (LeoDog896) [reply types/rollup-plugin-less/index.d.ts:7]: Completely forgot about this and came back -- it seems that less requires HTMLLinkElement which is why I added jsdom.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@LeoDog896 thx!

**Pull request merged**